docs/IE_DG/Model_caching_overview.md

```diff
@@ -0,0 +1,82 @@
+# Model Caching Overview {#openvino_docs_IE_DG_Model_caching_overview}
+
+## Introduction
+
+As described in [Inference Engine Introduction](inference_engine_intro.md) the common application flow contains the following steps:
+
+1. **Create Inference Engine Core object**
+
+2. **Read the Intermediate Representation** - Read an Intermediate Representation file into an object of the `InferenceEngine::CNNNetwork`
+
+3. **Prepare inputs and outputs**
+
+4. **Set configuration** Pass device specific loading configurations to the device
+
+5. **Compile and Load Network to device** - Use the `InferenceEngine::Core::LoadNetwork()` method with specific device
+
+6. **Set input data**
+
+7. **Execute**
+
+Step #5 can potentially perform several time-consuming device-specific optimizations and network compilations,
+and such delays can lead to bad user experience on application startup. To avoid this, some devices offer
+Import/Export network capability, and it is possible to either use [Compile tool](../../inference-engine/tools/compile_tool/README.md)
+or enable model caching to export compiled network automatically. Reusing cached networks can significantly reduce load network time
+
+
+## Set "CACHE_DIR" config option to enable model caching
+
+To enable model caching application shall specify folder where to store cached blobs. It can be done like this
+
+```cpp
+InferenceEngine::Core ie;                                 // Step 1: create Inference engine object
+ie.SetConfig({{CONFIG_KEY(CACHE_DIR), "myCacheFolder"}}); // Step 1b: Enable caching
+auto cnnNet = ie.ReadNetwork(<modelFile>);                // Step 2: ReadNetwork
+...                                                       // Step 3: Prepare inputs/outputs
+...                                                       // Step 4: Set device configuration
+ie.LoadNetwork(cnnNet, <device>, <deviceConfig>);         // Step 5: LoadNetwork
+```
+With this code, if device supports Import/Export network capability, cached blob will be automatically created inside `myCacheFolder` folder
+CACHE_DIR config is set to Core object, so if device doesn't support Import/Export capability, cache will just not be created and no error will be thrown
+
+Depending on your device, total time for loading network on application startup can be significantly reduced.
+Please also note that very first LoadNetwork (when cache is not yet created) will take slightly longer time to 'export' compiled blob into a cache file
+![caching_enabled]
+
+## Even faster: use LoadNetwork(\<modelName\>)
```

Comment:
modelName -> modelFileName? to be more specific

Reply:
Renamed to modelPath as used in code snippets